src/paginate.ts

```diff
@@ -362,9 +370,6 @@ export async function paginate<T extends ObjectLiteral>(
 
     ;[items, totalItems] = await queryBuilder.getManyAndCount()
```

Comment:
does this still execute 2 queries ? (1) count and (2) find?

should be then just using `getMany()`. no need to count if you have no limit.